frappe/public/js/frappe/form/form.js

```diff
@@ -1228,6 +1228,10 @@ frappe.ui.form.Form = class FrappeForm {
 		frappe.set_route("print", this.doctype, this.doc.name);
 	}
 
+	show_audit_trail() {
+		frappe.set_route("audit-trail");
+	}
```

Comment:
Not sure why it's circular like this :grimacing:

Reply:
Intended to auto-set the filters using the form arguments but since it is not supported for Single DocTypes (https://github.com/frappe/frappe/issues/23061), ended up missing this detail. Will create a PR for changes.